src/ocp_postprocess/ip_rename/etcd_rename.rs

```diff
@@ -448,6 +449,16 @@ pub(crate) async fn fix_networks_cluster(etcd_client: &Arc<InMemoryK8sEtcd>, ip:
     Ok(())
 }
 
+pub(crate) async fn fix_etcd_member(etcd_client: &Arc<InMemoryK8sEtcd>, ip: &str) -> Result<()> {
+    let mut update = format!("https://{}:2380", ip).to_string();
+    if !ip.parse::<Ipv6Addr>().is_err() {
```

Comment:
positive easier to read than double negative

```suggestion
    if ip.parse::<Ipv6Addr>().is_ok() {
```